### PR TITLE
Suggestion: comment should include complete file

### PR DESCRIPTION
**Pull request by bkadatz**
For people who jump straight into the example without thoroughly reading the documentation, providing the complete file within the comment clarifies the YAML's overall structure.

### CODE REVIEW

File: cfg_template.yml [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
+# The order of branches is important - it defines what will be merged (or rebased) into what.
+#
+# The train identifier ("fix important things") is only there for readability and it can be any string. We
+# suggest you use a short description explaining what the PR train is for.
+#
+# The "combined" branch is optional. If you include it, pr-train will create a "combined" branch for you.
+#
+# This file should end up looking like:
+#
+# trains:
```
Comment (realyze) [inline cfg_template.yml:14]: Would this perhaps be confusing in that people would just uncomment this section and have `trains:` in the file twice?

**Comment by realyze** [comment]
@bkadatz Thanks for the suggestions and sorry for the delay!

File: cfg_template.yml (hunk 1 above)
Comment (bkadatz) [reply cfg_template.yml:14]: What confused me is that I thought the train identifier (`fix important things:`) was equivalent to the `trains:` line. But that's definitely user error on my part as the other docs show a complete example with both. And it's probably just me so go ahead and close this. :)

Comment (realyze) [reply cfg_template.yml:14]: Right, I hear what you're saying. I think it's probably safer to _not_ duplicate the `trains:` label as I can imagine that could easily cause errors & confusion.

So I'll close this - but I appreciate the PR and thanks for your effort! 🙇

**Pull request closed without merging**